src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs

```diff
@@ -80,6 +84,7 @@ protected override string AddFromMagnetLink(RemoteEpisode remoteEpisode, string
             var isRecentEpisode = remoteEpisode.IsRecentEpisode();
             var moveToTop = (isRecentEpisode && Settings.RecentTvPriority == (int)QBittorrentPriority.First) || (!isRecentEpisode && Settings.OlderTvPriority == (int)QBittorrentPriority.First);
             var forceStart = (QBittorrentState)Settings.InitialState == QBittorrentState.ForceStart;
+            var propagateTags = Settings.PropagateTags;
```

Comment:
No need to store this in a separate variable since it's only used once and is a simple boolean.